.gitignore

```diff
@@ -42,3 +42,4 @@ storage/
 yarn-debug.log*
 
 /config/master.key
+/config/clave/
```

Comment:
what are in this folder?

Reply:
see the ending of the Intallation section: https://github.com/CodiTramuntana/decidim-module-clave#installation